### ISSUE TITLE
Holistic audit of current dashboard

### ISSUE THREAD

**Issue by thisisdano** [18F/cg-design#7]
Our recent work on the dashboard has been very feature focused. But what is the current state of the dashboard — both in broad strokes and in more detail? What remains awkward? What still needs to be done to create or enhance a coherent product vision?

A dashboard audit will look at every major screen and provide feedback on what needs work — whether that's visual refinement, copy, interaction, or more structural issues.

If we can create a clear-eyed view of where we are now, we can better plan for future work.

Initial audit feedback is on an InVision prototype: https://gsa.invisionapp.com/share/RW8IGQS78

Next steps TBD.

Acceptance criteria: 
- [ ] Summary documentation written and available for full program team via Google Docs or Github

**Comment by suprenant** [comment]
@thisisdano I made a minor update to make the acceptance criteria clear. Let's get into the habit of doing this for all of our stories!

**Comment by thisisdano** [comment]
@suprenant sounds good — I didn't have a sense of acceptance criteria when creating the issue — especially wrt deliverables and shareout.

**Comment by thisisdano** [comment]
IN GENERAL — I think there's no SOP for deliverables/shareout on things like this (and interview feedback — that kind of stuff). And there should be.

**Comment by suprenant** [comment]
@thisisdano totally fine! Whenever we take on work, we should think two things: to whom is this delivering what value, and how can we share/demonstrate this when we're done? 

Given that we're a distributed team with lots of moving pieces, sharing our learnings for the latter is especially crucial. 

If we don't know the answers to either item, that's okay, but then let's talk them through so we can come up with something! And if we can't, then we can always prioritize other work.

**Comment by suprenant** [comment]
@thisisdano as discussed, let's timebox this so this is wrapped by COB Tuesday. Does that work?

**Comment by thisisdano** [comment]
## Goal

To identify weakness and ways forward with the current cloud.gov dashboard so the design team has a cleareyed view of its present challenges and a framework for success as cloud.gov attempts to scale to a broader base of paying customers. 

## Preamble

I completed this somewhat gloomy report and analysis prior to today's conversations about our upcoming hard stop. What I detail here are the problems I see with building consensus to justify certain "higher level" design decisions (IA, pattern library, user journeys, design principles) over immediate triage and feature adds. This difficulty makes it hard to prioritize the kind of metadesign work and polish work that makes a product strong and coherent — to justify the kinds of heavy lifts that might not make as clear marketing bullet points as features. Going forward, I suggested backfilling our product documentation with the kinds of documentation of vision and principle that would help justify these needs as business needs and product needs — to help us move forward now, and to provide a solid framework to evaluate what we've done and continue to develop.

Given our new time/resource constraints, we might see our priorities shift without the need for such a framework. It's still pretty important for the continued health of the product that such a framework exists — it's a critical consensus-building and decision-making tool for the marketing and product side as much as for the dashboard — and, as a designer, I wish we'd worked on this first, rather than retroactively. I still plan to make some progress over our IP sprint.

## tl;dr

The current cloud.gov dashboard suffered from a premature rollout and a lack of preliminary pre-launch design planning, resulting in uncoordinated execution thereafter. The current dashboard’s state of inconsistent patterns, unreliable hierarchy, clumsy navigation, and wobbly tone doesn’t set clear expectations for the user, nor does it clearly reflect or inspire the kind of confidence in the platform we desire. These weaknesses are a result of incomplete organizing principles. I recommend developing a set of clear guiding principles as a framework for coherent design and ongoing decision making.

Full report here: https://docs.google.com/a/gsa.gov/document/d/1k79Pp72xRWXgNvbVOnS8aBAQ-ckNCXbkwXyAVW53O0s/edit?usp=sharing

**Comment by suprenant** [comment]
Looks good! Will look forward to seeing what we can pull from this to help us this cycle

**Comment by suprenant** [comment]
@thisisdano as we process this as a team and decide how we want to act on it, please remember that we'll want to demo this in some capacity on Wednesday's sprint review – possibly a one-slide summary of the high-level recommendations with a link to the document? 

cc @vz3

**Comment by suprenant** [comment]
Talked to @vz3 – thinking we'll include this in the WIP section for now for the demo

**Comment by thisisdano** [comment]
@suprenant Cool — I've been thinking a lot more about this, and I would like to connect it to the IP sprint, and to some explicit groundwork for the next PI. And the more I've thought about it, I think it actually connects in a meaningful way to our theme name. (Which I now consider it important to change, as an expression of explicit product/design principles.) I have more thoughts about this, tk. 

Further deliverables for this could include specific next steps. I/We could create these over the next sprint and present at the end — then I'd work toward some of those goals in IP.

More to come.

**Comment by suprenant** [comment]
@thisisdano sounds good. Let's give the team a little time to chew on what's here, and we can work together to outline and prioritize our next steps. I'll defer to @vz3, but the IP sprint does seem like a good time for this

**Comment by thisisdano** [comment]
This is a good place to start when thinking about design process/deliverables: https://github.com/18F/Design-Wiki/wiki
